pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/PulsarAdminImpl.java

```diff
@@ -215,13 +212,35 @@ public PulsarAdminImpl(String serviceUrl,
         }
     }
 
+    private Authentication determineAuth(ClientConfigurationData conf) throws PulsarClientException {
+        if (conf == null) {
+            return new AuthenticationDisabled();
+        }
+        maybeSetAuthOnConf(conf);
+        return conf.getAuthentication();
+    }
+
+    private void maybeSetAuthOnConf(ClientConfigurationData conf) throws PulsarClientException {
+        if (StringUtils.isBlank(conf.getAuthPluginClassName())
```

Comment:
This is functionally the same thing the regular client does [here](https://github.com/apache/pulsar/blob/a2c15340d3a73401b57574a4032ca375bde502bc/pulsar-client/src/main/java/org/apache/pulsar/client/impl/PulsarClientImpl.java#L192).